code/game/objects/structures/fluff.dm

```diff
@@ -285,3 +285,43 @@
 /obj/structure/fluff/tram_rail/anchor
 	name = "tram rail anchor"
 	icon_state = "anchor"
+
+/obj/structure/fluff/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "desk_bell"
+	pass_flags = PASSTABLE
+	layer = OBJ_LAYER
+	deconstructible = FALSE
+	anchored = FALSE
+	/// A cooldown timer so people don't pull out their autoclickers
```

Comment:
no, remove this

Reply:
what, the timer or the stuff above the comment?

Reply:
If you were talking about the timer, I made a variant with no timer. I don't know where it'd go, so for now it's just an adminspawn thing.